lib/containers/container_images.pm

```diff
@@ -195,8 +221,11 @@ sub test_zypper_on_container {
         # Remove it
         assert_script_run("$runtime rm refreshed", 120);
         # Verify the image works
-        assert_script_run("$runtime run --rm refreshed-image sh -c 'zypper -v ref | grep \"All repositories have been refreshed\"'", 120);
+        my $check = 'grep \"All repositories have been refreshed\"';
+        my $cmd   = "$runtime run --rm refreshed-image";
+        assert_script_run($runtime =~ /docker/ ? "$cmd sh -c ' zypper -v ref | $check'" : "$cmd zypper -v ref | $check", 120);
```

Comment:
What is this change needed for?

Reply:
the problem is visible here http://aquarius.suse.cz/tests/4741#step/rootless_podman/100. it needs investigation but i didnt have the time so far.

Reply:
I would vote to do this investigation before merging this

Reply:
i would not to have the time for now. but i promise to dig into it soon

Reply:
It works here, the linked failure seems to be connected to systemd logging somehow...

If it for some reason that error is reproducible, that's likely a product bug and shouldn't blindly be worked around.

Reply:
i tried on a vm locally and i didnt reproduce it. i will try to debug again on monday (or during the weekend) on OpenQA

Reply:
i checked the problem on OpenQA and i can say that it is not a product bug for sure. I reverted the change and run some VR again.